sdk/src/Services/DynamoDBv2/Custom/Conversion/SchemaV1.cs

```diff
@@ -30,227 +31,323 @@ namespace Amazon.DynamoDBv2
 {
     #region Basic converters
 
-#if NET8_0_OR_GREATER
-    [System.Diagnostics.CodeAnalysis.RequiresUnreferencedCode(Amazon.DynamoDBv2.Custom.Internal.InternalConstants.RequiresUnreferencedCodeMessage)]
-#endif
     internal class ByteConverterV1 : Converter<byte>
     {
+        public override IEnumerable<Type> GetTargetTypes()
```

Comment:
These are the methods that were added after removing parent override that was using unbounded reflection.